src/Artsy/Analytics/Schema/Values.ts

```diff
@@ -54,6 +54,8 @@ export enum ActionType {
    */
   ClickedReadMore = "Clicked read more",
 
+  ClickedCommercialFilter = "Clicked commercial filter params",
```

Comment:
This is worth double checking with @cavvia about the desired naming, this differs slightly from what's used for the filter on /collect: https://github.com/artsy/reaction/blob/fd1922d49906c2197834bdc69eebb25a62860007/src/Apps/Collect/FilterState.tsx#L171